### ISSUE TITLE
Re-write of Boston Light Swim

### ISSUE THREAD

**Issue by ocefpaf** [ioos/notebooks_demos#25]
@rsignell-usgs as I am adapting this notebook for bokeh plots I realized that the many serializations peform (`iris`, `netcdf`, `csv`, `Dataframe`, etc) can be simplified with a better metadata preserving model that "speaks pandas" (`xarray`).

So I will finish a draft tomorrow but wanted to ping you before completion to get your opinion on this.

**Comment by rsignell-usgs** [comment]
Sounds good to me.  Any downsides?

**Comment by ocefpaf** [comment]
> Any downsides?

Non-CF "free" compliant netCDF files. We need to be careful to ensure compliance. However, I had another idea and I am working on it now. I will try to add an extra Auxiliary coordinate with the stations long names and another one for the station code. That way we have all the information in one netCDF CF compliant file.

**Comment by ocefpaf** [comment]
I changed the strategy in #28. I still use `iris`, but I no longer add the stations as a coordinate. Instead they are independent phenomena (valid CF). That way we can add as many auxiliary coordinates, with the remaining metadata, we as like. I also added convenience functions to go from iris cubes to a list of pandas Series with the metadata.

xarray is still tempting but I do not want to keep track of CF compliance and the temptation to break it (and make things easier) is high with xarray.
